app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -114,6 +116,9 @@ public class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPage
                 routePresenter?.onInstructionSelected(instruction)
             }
         }
+        notificationCreator?.createNewNotification(
```

Comment:
When testing the notification using a mock route, the instruction text shown in the notification always lagged one behind the current instruction shown in the view pager.

It seems to work better if this logic is moved into `setCurrentInstruction(index: Int)`

Also, `route?.getCurrentInstruction()` is not always the same as the current instruction that is displayed to the user (since we give the user advance warning of upcoming maneuvers). Better to use `route?.getRouteInstructions()?.get(index)` to determine which instruction to display.